nixos/modules/security/apparmor/includes.nix

```diff
@@ -0,0 +1,301 @@
+{ config, lib, pkgs, ... }:
+let
+  inherit (builtins) attrNames hasAttr isAttrs;
+  inherit (lib) getLib;
+  inherit (config.environment) etc;
+  etcRule = arg:
```

Comment:
I can read what this is doing but do you mind adding a comment here? You probably wrote this function as it is "abstracting" some complexity away from the code further down this file. Now I just have to grok what that complexity is.

Reply:
Sure.